Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/CardHeader.cpp

```diff
@@ -21,12 +21,43 @@
 #include <QPushButton>
 #include <QStyle>
 #include <QTimer>
+#include <QPainter>
 
 namespace AzQtComponents
 {
     static QString g_containerCardHeaderClass = QStringLiteral("ContainerCardHeader");
     static QString g_sectionCardHeaderClass = QStringLiteral("SectionCardHeader");
 
+    namespace Internal
+    {
+        ClickableIconLabel::ClickableIconLabel(QWidget* parent)
+            : QLabel(parent)
+        {
+        }
+
+        void ClickableIconLabel::SetClickable(bool clickable)
+        {
+            m_clickable = clickable;
+        }
+
+        bool ClickableIconLabel::GetClickable() const
+        {
+            return m_clickable;
+        }
+
+        void ClickableIconLabel::mouseReleaseEvent(QMouseEvent* event)
+        {
+            if (m_clickable)
+            {
+                emit clicked(event->globalPos());
+            }
+            else
+            {
+                event->ignore();
+            }
+        }
+    }
+
```

Comment:
Instead of making a clickable label, you could just change `m_iconLabel` to be a `QToolButton`, which would give you both an icon (pixmap) and be clickable.